Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -164,7 +169,11 @@ fi
 
 # Check for graphical package for MAVProxy
 if [[ $SKIP_AP_GRAPHIC_ENV -ne 1 ]]; then
-  if [ ${RELEASE_CODENAME} == 'focal' ]; then
+  if [ ${RELEASE_CODENAME} == 'groovy' ]; then
```

Comment:
why not [ ${RELEASE_CODENAME} == 'focal' ] || [ ${RELEASE_CODENAME} == 'groovy' ] ?

Reply:
The pattern in here seems to be going to per-release clauses.